src/main/java/dev/dfonline/codeclient/Commands.java

```diff
@@ -610,7 +610,7 @@ else if(field.getType().isEnum()) {
 
         dispatcher.register(literal("ping").executes(context -> {
             assert CodeClient.MC.player != null;
-            int ping = Objects.requireNonNull(CodeClient.MC.player.networkHandler.getPlayerListEntry(CodeClient.MC.player.getUuid())).getLatency();
+            int ping = CodeClient.MC.player.networkHandler.getPlayerListEntry(CodeClient.MC.player.getUuid()).getLatency();
```

Comment:
turn the Objects.requireNonNull check you removed and `assert CodeClient.MC.player != null` to a `if (CodeClient.MC.player == null || X) return;` so its cleaner and doesnt throw an error in some weird state where they are null

Reply:
i actually did not see assert there lol